drivers/gpio/gpio_nxp_s32.c

```diff
@@ -66,13 +75,24 @@ struct gpio_nxp_s32_data {
 #endif
 };
 
+static inline uint16_t reverse_bits_16(uint16_t value)
```

Comment:
Not sure if zephyr exposes inline ARM assembly but you could swap this with
https://developer.arm.com/documentation/ddi0403/d/Application-Level-Architecture/Instruction-Details/Alphabetical-list-of-ARMv7-M-Thumb-instructions/RBIT?lang=en

Reply:
changed to `__RBIT` from cmsis